src/com/lilithsthrone/rendering/Pattern.java

```diff
@@ -0,0 +1,111 @@
+package com.lilithsthrone.rendering;
+
+import java.io.File;
+import java.io.FilenameFilter;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Pattern {
+	
+	private static Map<String, Pattern> allPatterns;
+	
+	private String name;
+	private String SVGString; // TODO ADD THIS
+	
+	static {
```

Comment:
this sort of thing really ought to be forbidden at one point.

... but it is how we do stuff so whatever.

Reply:
Loading patterns during initialization of runtime with a static constructor? Is it really that bad? 

I mean in my own project I prefer to instead have singletons with Init() method or something like that... Also, similar usage was in the game already somewhere I think.

Reply:
I agree that this isn't on you to fix; with that in mind, I'd like to explain:

If I recall correctly, static initializers run when the class is first loaded. This means you have no real control over when this is done, just that you get the guarantee that it is done before it is used.

Because it is executed when first used, you might have a slowdown the first time someone starts a new game or loads a savefile. This in and of itself isn't bad...

If you had a bug in here, however, the game would crash at a "random" time. And the stack trace... I don't know what the stack trace would look like. I do know that there's no way to try-catch exceptions from static code blocks.